index.js

```diff
@@ -1,10 +1,10 @@
-if (!Array.prototype.smoosh) {
+try {[].smooch()} catch(_) {
   Array.prototype.smoosh = function smoosh() {
     return this[0];
   };
 }
 
-if (!Array.prototype.smooshMap) {
+try {[].smoochMap(function(){return []})} catch(_) {
```

Comment:
Same issue here - `smoochMap` => `smooshMap`.

Also, why is`smooshMap` doing `return []`? The `smooshMap` spec is perfectly happy if `undefined` is returned, so there should be no need for a function body at all.